site-src/contributing/enhancement-requests.md

```diff
@@ -1,34 +1,40 @@
 # Enhancement Tracking and Backlog
 
-Inspired by [Kubernetes enhancements](https://github.com/kubernetes/enhancements), service-api's
-provides a process for introducing new functionality or considerable changes to the project. The
-enhancement process will evolve over time as the project matures.
+Inspired by [Kubernetes enhancements][enhance], service-api's provides a
+process for introducing new functionality or considerable changes to the
+project. The enhancement process will evolve over time as the project matures.
 
-Enhancements provides the basis of a community roadmap. Enhancements may be filed by anyone, but
-require approval from a maintainer to accept the enhancement into the project.
+[enhance]: https://github.com/kubernetes/enhancements
+
+Enhancements provides the basis of a community roadmap. Enhancements may be
+filed by anyone, but require approval from a maintainer to accept the
+enhancement into the project.
 
 ## Quick start
 
-1. Create an [Issue](https://github.com/kubernetes-sigs/gateway-api/issues/new/choose) and select
-"Enhancement Request".
-2. Follow the instructions in the enhancement request template and submit the Issue.
+1. Create an [Issue][issue] and select "Enhancement Request".
+2. Follow the instructions in the enhancement request template and submit the
+   Issue.
+3. (depending on size of change) Start a [draft GEP proposal][gep]
```

Comment:
Nit: I'd recommend either just "Gateway Enhancement Proposal" here - "GEP Proposal" is repetitive.